app/src/main/java/com/spendster/MainActivity.java

```diff
@@ -1,13 +1,31 @@
 package com.spendster;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.spendster.presentation.WelcomeActivity;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        Button btnStartWelcomeActivity = findViewById(R.id.btnStartWelcomeActivity);
```

Comment:
@SimonMaha make Welcome screen activity to be a launcher in the manifest file. This way you will be able to remove this code for starting it.